ribs/emitters/_gaussian_emitter.py

```diff
@@ -56,15 +56,23 @@ def __init__(self,
 
         self._x0 = np.array(x0, dtype=archive.dtype)
         if self._x0.ndim != 1:
-            raise ValueError(
-                f"x0 has shape {self._x0.shape}, should be 1-dimensional.")
+            raise ValueError(f"x0 has shape {self._x0.shape}, should be"
+                             f"1-dimensional.")
 
-        self._sigma = archive.dtype(sigma) if isinstance(
-            sigma,
-            (float, np.floating)) else np.array(sigma, dtype=archive.dtype)
-        self._sigma0 = self._sigma if sigma0 is None else (
-            archive.dtype(sigma0) if isinstance(sigma0, (
-                float, np.floating)) else np.array(sigma0, dtype=archive.dtype))
+        if isinstance(sigma, (float, np.floating)):
```

Comment:
We should just use `np.array` here regardless since `np.array` can take in scalars.